projects/microchip/curiosity_pic32mzef/mplab/aws_demos/nbproject/project.xml

```diff
@@ -12,7 +12,19 @@
             <header-extensions>h</header-extensions>
             <asminc-extensions/>
             <sourceEncoding>ISO-8859-1</sourceEncoding>
-            <make-dep-projects/>
+            <make-dep-projects>
```

Comment:
Do we need to modify aws_tests project as well ?

Reply:
No , we have decided to keep the binary linked for tests project.